t/opcode-coverage.neon

```diff
@@ -0,0 +1,594 @@
+FUNCTION callf()
+    callp()
+END FUNCTION
+
+#    CALLMF,     // call module function
+FUNCTION callmf(): Number
+    RETURN module.function(1)
+END FUNCTION
+
+#    CALLI,      // call indirect
+FUNCTION calli()
+    FUNCTION life()
+    END FUNCTION
+    VAR pLife: FUNCTION() := life
+    pLife()
+END FUNCTION
+
+#    JUMP,       // unconditional jump
+FUNCTION jump()
+    IF 1 = 2 THEN
+    END IF
+END FUNCTION
+
+#    JF,         // jump if false
+FUNCTION jf()
+    FOR i := 1 TO 2 DO
+    END FOR
+END FUNCTION
+
+#    JT,         // jump if true
+FUNCTION jt():Boolean
+    RETURN TRUE OR FALSE
+END FUNCTION
+
+#    JFCHAIN,    // jump and drop next if false
+FUNCTION jfchain(): Boolean
+    RETURN 1 < 2 < 3
+END FUNCTION
+
+#    DUP,        // duplicate
+FUNCTION dup():Boolean
+    VAR x: Boolean := TRUE OR FALSE
+    RETURN x
+END FUNCTION
+
+#    DUPX1,      // duplicate under second value
+FUNCTION dupx1()
+    VAR x: POINTER TO class
+    x := NEW class(Field WITH 42)
+END FUNCTION
+
+#    DROP,       // drop
+FUNCTION drop():Boolean
+    VAR x: Boolean := TRUE OR FALSE
+    RETURN x
+END FUNCTION
+
+#    RET,        // return
+FUNCTION ret()
+END FUNCTION
+
+#    CONSA,      // construct array
+FUNCTION consa(): Array<Number>
+    RETURN [42]
+END FUNCTION
+
+#    CONSD,      // construct dictionary
+FUNCTION consd(): Dictionary<Number>
+    RETURN {"FortyTwo": 42}
+END FUNCTION
+
+#    EXCEPT,     // throw exception
+FUNCTION except()
+    RAISE TestException
+END FUNCTION
+
+#    ALLOC,      // allocate record
+FUNCTION alloc()
+    VAR x: POINTER TO class
+    x := NEW class(Field WITH 42)
+END FUNCTION
+
+#    PUSHNIL,    // push nil pointer
+FUNCTION pushnil(): POINTER TO class
+    VAR x: POINTER TO class := NIL
+    RETURN x
+END FUNCTION
+
+#    RESETC,     // reset cell
+FUNCTION resetc()
+    VAR x: Number
+    x := 1
+END FUNCTION
+
+#    PUSHPEG,    // push pointer to external global
+FUNCTION pushpeg()
+    # This requires the REPL to implement.
+END FUNCTION
+
+#    JUMPTBL,    // jump table
+FUNCTION jumptbl(): Number
+    CASE(1)
+        WHEN (0) DO RETURN 0
+    END CASE
+    RETURN -1
+END FUNCTION
+
+#    CALLX,      // call extension
+FUNCTION callx(): Boolean
+    DECLARE EXTENSION FUNCTION funcBoolean(): Boolean
+    RETURN funcBoolean()
+END FUNCTION
+
+#    SWAP,       // swap two top stack elements
+FUNCTION swap()
+    VAR x: Number := 1
+    INC x
+END FUNCTION
+
+#    DROPN,      // drop element n
+FUNCTION dropn()
+    DECLARE EXTENSION FUNCTION funcDrop(OUT a: Number)
+    VAR a: Number := -1
+    funcDrop(OUT a)
+END FUNCTION
+
+#    PUSHFP,     // push function pointer
+FUNCTION pushfp()
+    VAR ptr: FUNCTION()
+    ptr := pushfp
+END FUNCTION
+
+#    CALLV,      // call virtual
+FUNCTION callv()
+    LET i: POINTER TO Interface := NEW class
+    i->f()
+END FUNCTION
+
+#    PUSHCI,     // push class info
+FUNCTION pushci()
+    VAR x: POINTER TO class
+    x := NEW class(Field WITH 42)
+END FUNCTION
```

Comment:
Where's the calls to all these functions? This should be a runnable sample that actually executes all the opcodes it says it does.

Reply:
Added calls to all functions that _can_ be called.  Looking into the others.